src/Elm/OperatorTable.elm

```diff
@@ -0,0 +1,188 @@
+          , precedence = Node Range.emptyRange 7
+          , operator = Node Range.emptyRange "//"
+          , function = Node Range.emptyRange "idiv"
+          }
+        , -- infix right 8 (^)  = pow
+          { direction = Node Range.emptyRange Right
+          , precedence = Node Range.emptyRange 8
+          , operator = Node Range.emptyRange "^"
+          , function = Node Range.emptyRange "pow"
+          }
+        , -- infix left  9 (<<) = composeL
+          { direction = Node Range.emptyRange Left
+          , precedence = Node Range.emptyRange 9
+          , operator = Node Range.emptyRange "<<"
+          , function = Node Range.emptyRange "composeL"
+          }
+        , -- infix right 9 (>>) = composeR
+          { direction = Node Range.emptyRange Right
+          , precedence = Node Range.emptyRange 9
+          , operator = Node Range.emptyRange ">>"
+          , function = Node Range.emptyRange "composeR"
+          }
+        , -- List module
+          -- infix right 5 (::) = cons
+          { direction = Node Range.emptyRange Right
+          , precedence = Node Range.emptyRange 5
+          , operator = Node Range.emptyRange "::"
+          , function = Node Range.emptyRange "cons"
+          }
+        , -- elm/url
+          -- Url.Parser module
+          -- infix right 7 (</>) = slash
+          { direction = Node Range.emptyRange Right
+          , precedence = Node Range.emptyRange 7
+          , operator = Node Range.emptyRange "</>"
+          , function = Node Range.emptyRange "slash"
+          }
+        , -- infix left  8 (<?>) = questionMark
+          { direction = Node Range.emptyRange Left
+          , precedence = Node Range.emptyRange 8
+          , operator = Node Range.emptyRange "<?>"
+          , function = Node Range.emptyRange "questionMark"
+          }
+        , -- elm/parser
+          -- Parser module
+          -- infix left 5 (|=) = keeper
+          { direction = Node Range.emptyRange Left
+          , precedence = Node Range.emptyRange 5
+          , operator = Node Range.emptyRange "|="
+          , function = Node Range.emptyRange "keeper"
+          }
+        , -- infix left 6 (|.) = ignorer
+          { direction = Node Range.emptyRange Left
+          , precedence = Node Range.emptyRange 6
+          , operator = Node Range.emptyRange "|."
+          , function = Node Range.emptyRange "ignorer"
+          }
+        , -- Parser.Advanced module
+          -- infix left 5 (|=) = keeper
+          { direction = Node Range.emptyRange Left
+          , precedence = Node Range.emptyRange 5
+          , operator = Node Range.emptyRange "|="
+          , function = Node Range.emptyRange "keeper"
+          }
+        , -- infix left 6 (|.) = ignorer
+          { direction = Node Range.emptyRange Left
+          , precedence = Node Range.emptyRange 6
+          , operator = Node Range.emptyRange "|."
+          , function = Node Range.emptyRange "ignorer"
```

Comment:
Should `|=` and `|.` be listed here twice?

Reply:
No, that's indeed unnecessary :+1: